src/smc_sagews/smc_sagews/tests/test_sagews_modes.py

```diff
@@ -46,3 +47,64 @@ def test_sh_autocomplete_02(self, test_id, sagews):
         assert mesg['event'] == "introspect_completions"
         assert mesg['completions'] == ["AR29"]
         assert mesg['target'] == "$TESTV"
+
+class TestShDefaultMode:
+    def test_start_sh(self, exec2):
+        exec2("%default_mode sh")
+    def test_start_sh2(self, exec2):
+        exec2("pwd")
+
+    def test_single_line(self, exec2):
+        exec2("pwd\n", html_pattern=">/")
+
+    def test_multiline(self, exec2):
+        exec2("FOO=hello\necho $FOO", html_pattern="hello")
+
+    def test_date(self, exec2):
+        exec2("date +%Y-%m-%d", html_pattern = '\d{4}-\d{2}-\d{2}')
+
+    def test_capture_sh_01(self, exec2):
+        exec2("%capture(stdout='output')\nuptime")
```

Comment:
I'm not 100% sure about this test. I tried running these examples directly in a sagews, and what did work for me is:

```
%capture(stdout='output')
%sh
uptime
```
and then
```
%sage
output
```

Just like in the R case below, there is an additional `%sh` like there is an additional `%r`  in the `%capture`. 

Or is there a default set, which I'm not seeing here? In any case, the test did work, but in my sagews the `output` was empty.